gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ApiLogsMapper.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.management.v2.rest.mapper;
+
+import io.gravitee.rest.api.management.v2.rest.model.ApiLog;
+import io.gravitee.rest.api.model.v4.log.ConnectionLogModel;
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Mapper(uses = { ApplicationMapper.class, DateMapper.class, PlanMapper.class })
+public interface ApiLogsMapper {
+    Logger logger = LoggerFactory.getLogger(ApiLogsMapper.class);
+    ApiLogsMapper INSTANCE = Mappers.getMapper(ApiLogsMapper.class);
+
+    @Mapping(source = "timestamp", target = "timestamp", qualifiedByName = "mapTimestamp")
```

Comment:
I think we don't need to add this annotation. To be confirmed, but I think that MapStruct will find the right method in the dependent mappers

Reply:
It seems the annotation is required

```
java.lang.RuntimeException: java.text.ParseException: Unparseable date: "2020-02-01T20:00:00.00Z"
	at io.gravitee.rest.api.management.v2.rest.mapper.ApiLogsMapperImpl.map(ApiLogsMapperImpl.java:38)
```